src/OrchardCore.Modules/OrchardCore.Media.Azure/Services/AzureBlobStorageCacheOptionsConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.Extensions.Options;
+using SixLabors.ImageSharp.Web.Caching.Azure;
+
+namespace OrchardCore.Media.Azure.Services;
+
+// Configuration for ImageSharp's own configuration object. We just pass the settings from the Orchard Core
+// configuration.
+internal class AzureBlobStorageCacheOptionsConfiguration : IConfigureOptions<AzureBlobStorageCacheOptions>
+{
+    private readonly IOptions<ImageSharpImageCacheOptions> _options;
```

Comment:
Maybe localize the instance only without the `IOptions<>` like we do everywhere else. then in the constructor `options.Value`

private readonly ImageSharpImageCacheOptions _cacheOptions;